### DISCUSSION TITLE
Motions like $d, ^d?

### DISCUSSION THREAD

**Discussion by gekoke** [qutebrowser/qutebrowser#7127] [Q&A] [answered]
Is there a way to delete all tabs to the end, or all tabs to the beginning  ($d, ^d basically). I couldn't find anything like that in the when browsing the commands (though I did find :tab-only).

My use case is that I usually keep my important tabs that I don't want to close on the left, whilst tabs that I don't really care about accumulate on the right. In brave + vimium, there is an action (that I've bound to $d) that when invoked deletes all tabs to the right, which I often use to close the unimportant tabs.

**Comment by toofar** [comment, marked as answer]
That would be `tab-only --prev` and `tab-only --next`. https://qutebrowser.org/doc/help/commands.html#tab-only

**Reply by gekoke** [reply]
That's it - thanks!
